sdk/browser/test/unit/AffinityWallet.test.ts

```diff
@@ -104,64 +86,21 @@ describe('AffinityWallet', () => {
   })
 
   describe('#deleteCredential', () => {
-    it('should throw if both parameters are given', async () => {
-      const affinityWallet = new AffinityWallet(walletPassword, encryptedSeed, sdkOptions)
-
-      let responseError
-
-      try {
-        await affinityWallet.deleteCredential('id', 1)
-      } catch (error) {
-        responseError = error
-      }
-
-      const { code } = responseError
-
-      expect(code).to.eql('COR-1')
-    })
+    it('should remove with given id', async () => {
+      sinon.stub(CommonNetworkMember.prototype as any, 'deleteCredentialById').resolves(undefined)
 
-    it('should throw if neither parameters are given', async () => {
       const affinityWallet = new AffinityWallet(walletPassword, encryptedSeed, sdkOptions)
 
-      let responseError
-
-      try {
-        await affinityWallet.deleteCredential(undefined, undefined)
-      } catch (error) {
-        responseError = error
-      }
-
-      const { code } = responseError
-
-      expect(code).to.eql('COR-1')
-    })
-
-    it('should remove with given index', async () => {
-      const credentialIndex = 1
-
-      sinon.stub(CommonNetworkMember.prototype as any, 'deleteCredentialByIndex').resolves(undefined)
-
-      const affinityWallet = new AffinityWallet(walletPassword, encryptedSeed, sdkOptions)
-
-      const result = await affinityWallet.deleteCredential(undefined, credentialIndex)
+      const result = await affinityWallet.deleteCredentialById(signedCredential.id)
```

Comment:
This test does not test anything now.

Maybe we should mock `WalletStorageService.deleteCredentialById` instead.

(But overall this is yet another example of how useless our unit tests are...)

Reply:
yes, many tests at platforms were testing code from mostly core while core didn't. Make sense to rethink way tests are testing. But this better to delete as for me